meson.build

```diff
@@ -129,7 +129,7 @@ libzstd = dependency(
 conf.set10('HAVE_LIBZSTD', libzstd.found())
 
 libcurl = dependency('libcurl',
-                     version : '>= 7.29.0')
+                     version : '>= 7.43.0')
```

Comment:
This is for `CURLPIPE_MULTIPLEX` and `CURLOPT_PIPEWAIT`. If you want to stick to `7.29` for some reason then we'll need some ifdefs on curl version I guess

Reply:
well, fedora30 is at 7.61 already, I think 7.43 should be totally fine too